spec/spec_helper.rb

```diff
@@ -1,3 +1,8 @@
+# Set default concurrency so we can check against it later. Must be set
+# before Vips.init sets concurrency to the default.
+DEFAULT_VIPS_CONCURRENCY = 5
+ENV["VIPS_CONCURRENCY"] = DEFAULT_VIPS_CONCURRENCY.to_s
+
```

Comment:
For predictable testing. Set our own VIPS_CONCURRENCY rather than let it default to CPU count.